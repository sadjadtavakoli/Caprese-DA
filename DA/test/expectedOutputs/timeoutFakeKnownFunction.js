
#5 function foo is called with variables args by timeoutFakeKnownFunction.js
#7 function foo entered with variables args from timeoutFakeKnownFunction.js
#8 function setTimeout is called with variables args by foo
#11 function setTimeout entered with variables args from foo
#11 function setTimeout exited with return values undefined to function foo
#7 function foo exited with return values undefined to function timeoutFakeKnownFunction.js
#end Execution