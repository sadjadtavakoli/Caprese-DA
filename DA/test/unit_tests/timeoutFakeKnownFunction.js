let bar = function() {
    console.log("Done!");
}

foo()

function foo(){
    setTimeout(bar, 10 * 1000)
}

function setTimeout(callback, time){
    // do nothing
}