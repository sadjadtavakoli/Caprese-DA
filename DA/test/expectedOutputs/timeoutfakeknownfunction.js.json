{"foo-test/unit_tests/timeoutfakeknownfunction.js-7-9":{"impacted":["t_1","t_8"]},"setTimeout-test/unit_tests/timeoutfakeknownfunction.js-11-13":{"impacted":["t_5"]},"keyMap":{"t_1":"test/unit_tests/timeoutfakeknownfunction.js","t_5":"foo-test/unit_tests/timeoutfakeknownfunction.js-7-9","t_8":"setTimeout-test/unit_tests/timeoutfakeknownfunction.js-11-13"}}